subprojects/logging/src/main/java/org/gradle/internal/featurelifecycle/LoggingDeprecatedFeatureHandler.java

```diff
@@ -30,6 +30,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(LoggingDeprecatedFeatureHandler.class);
     private static final String ELEMENT_PREFIX = "\tat ";
+    private static final String RUN_WITH_STACKTRACE_INFO = "\t(Run with --stacktrace to get the full stack trace of this deprecation warning.)";
```

Comment:
Actually both `--stacktrace` and `--full-stacktrace` say `This option renders stacktraces for deprecation warnings.` in the documentation.
Here I would stick with `--stacktrace` which is in sync with the usual `Run with --stacktrace option to get the stack trace.` you get when a build fails.